Node/core/lib/Session.js

```diff
@@ -407,6 +407,11 @@ var Session = (function (_super) {
     Session.prototype.sendBatch = function (done) {
         var _this = this;
         this.logger.log(this.dialogStack(), 'Session.sendBatch() sending ' + this.batch.length + ' message(s)');
+        if (this.batch.length == 0) {
+        		// No messages to send - Nothing to do.
+        		return;
```

Comment:
Please correct the spacing on these lines 411 and 412.

Reply:
I will do a new PR with both #3392 and #3398 including both Typescript and Javascript.